source/_drafts/Initial-Card-Designs-and-Evolution-of-Game-Rules.md

```diff
@@ -0,0 +1,40 @@
+title: Initial Card Designs and Evolution of Game Rules
+date: 2015-07-04 10:45:33
+category: design
+author: Chris Saylor
+tags:
+  - design
+  - cards
+  - rules
+thumbnail: /img/card-design-spell-v1.png
+---
+
+Over lunch, Steve and I had a _design sprint_ where the main topic was to discuss our different ideas about how the cards in _Witches and Warlocks_ were to be laid out and function.
+
+## Functionality and Flavor
+
+The card itself needs to convey important information to the player about what the card can do and how to play it, but we also wanted to leave some space to allow for flavor text and art, otherwise it would be boring!
+
+Our design decisions will always favor functionality first, so with that in mind, Priority of card information flows from top-to-bottom. Name, cost, effects, are at the top, while art and flavor text would be at the bottom.
+
+![](/img/card-design-spell-v1.png)
+
+> This is a sample of a spell card, name, effects, and cost at the top.
+
+## Card Ports
+
+Up until this point in order to construct and play a spell, the player could match like-element/artifact types a maximum of three spell cards together. One of the first concepts we discussed was the idea of cards having _ports_. This idea was to add a positional element to how the player can join spells together. Prior to this idea, you could match two spell cards together as long as they shared a common type (fire, sword, etc). Now, they have to match a type and a positional indicated by the port. Since we can control card linkage at the card level, this allowed us to remove the arbitrary three card limit of the spell.
```

Comment:
@cjsaylor I think this could use little bit of rewording, especially since we haven't really blogged about how anything works up to this point; new readers might be slightly confused.

But overall, I love the article.